lib/backend/k8s/resources/networkpolicy.go

```diff
@@ -398,16 +398,40 @@ func (npw *networkPolicyWatcher) processNPEvents() {
 		atomic.AddUint32(&npw.terminated, 1)
 	}()
 
-	var e api.WatchEvent
-	var isCRDEvent bool
-	var value interface{}
 	for {
+		var ok bool
```

Comment:
what's this doing here?  rebase include this commit accidentally?

Reply:
yeah, the rebase is a little wonky. sorry about that.